website/views.py

```diff
@@ -247,16 +248,16 @@ def newhome(request, template="new_home.html"):
     #     top_hunts = top_hunts.filter(is_published=True,result_published=False).order_by("-created")[:3]
 
 
-    context = {
-        "bugs": page_obj,
-        "bugs_screenshots" : bugs_screenshots,
-        # "server_url": request.build_absolute_uri('/'),
-        # "activities": activities,
-        # "hunts": Hunt.objects.exclude(txn_id__isnull=True)[:4],
-        "leaderboard": User.objects.filter(
-            points__created__month=datetime.now().month,
-            points__created__year=datetime.now().year,
-        )
+    # context = {
+    #     "bugs": bugs,
+    #     "bugs_screenshots" : bugs_screenshots,
+    #     # "server_url": request.build_absolute_uri('/'),
+    #     # "activities": activities,
```

Comment:
Please remove any commented code.

Reply:
done in commit 473919d8ca091a678234d2edd5dbedcbdb7855ba